spec/features/markdown_explanation_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'markdown explanation' do
+  before :each do
+    when_current_user_is :staff, integration: true
+    visit markdown_explanation_path
+  end
+  context 'markdown explanation renders properly' do
+    it 'renders bold properly' do
+      expect(find('li//strong', text: 'bold'))
```

Comment:
These aren't full expectations, because there's no matcher. It will still "fail", kinda, because Capybara will raise an exception when it cant be found. But I think these should be:

```ruby
expect(page).to have_selector('li//strong', text: 'bold')
```

Reply:
Yeah, I just want out about this, `('li//strong', text: 'bold')` and I wasn't sure if capybara failing to find element is a good way to fail a test. Your suggestion is way better.